src/main/java/oscar/login/jaas/BaseLoginModule.java

```diff
@@ -275,27 +274,27 @@ public void setSharedState(Map<String, ?> sharedState) {
         this.sharedState = sharedState;
     }
 
-    public Group getRolesGroup() {
+    public OscarGroup getRolesGroup() {
         return rolesGroup;
     }
 
-    public void setRolesGroup(Group rolesGroup) {
+    public void setRolesGroup(OscarGroup rolesGroup) {
```

Comment:
why the O word? can it be OpenoGroup?

Reply:
This is calling the OscarGroup class, the class was named as OscarGroup by the previous development team, I didn't try to change the class name.

Reply:
ok, if it is pre existing and must be used we can keep it.